### ISSUE TITLE
TypeScript support

### ISSUE THREAD

**Issue by andre347** [tableau/embedding-api-v3-guide#2]
Will v3 have TypeScript support? Is this on the roadmap?

**Comment by Kovner** [comment]
Yes it is. We are still working on whether it will be in the initial release though.

**Comment by florent-guinard** [comment]
I'm also looking forward to this enhancement: we're using the embedding API in a typescript application and having type definitions for it would be great for our code quality.

**Comment by erkobridee** [comment]
Is there any status on this topic? thanks o/

**Comment by anyoung-tableau** [comment]
Types are included in the NPM packages:

https://www.npmjs.com/package/@tableau/embedding-api
https://www.npmjs.com/package/@tableau/embedding-api-react
